colour/utilities/common.py

```diff
@@ -302,3 +305,48 @@ def is_integer(x):
     """
 
     return abs(x - round(x)) <= INTEGER_THRESHOLD
+
+
+def filter_kwargs(function, **kwargs):
```

Comment:
Yes, this is the purpose of that strange beast, it pretty much makes definitions with incompatible signatures, compatible.